src/TokenManagers/XboxTokenManager.js

```diff
@@ -70,13 +98,16 @@ class XboxTokenManager {
   async verifyTokens (relyingParty) {
     if (this.forceRefresh) return false
     const ut = await this.getCachedUserToken()
+    const tt = await this.getCachedTitleToken()
+    const dt = await this.getCachedDeviceToken()
     const xt = await this.getCachedXstsToken(relyingParty)
     debug('[xbl] have user, xsts', ut, xt)
-    if (xt.valid) return true
-    if (!ut.valid) return false
+
+    if (!ut.valid || (dt.exists && !dt.valid) || (tt.exists && !tt.valid)) return false
```

Comment:
Some of these checks should only be happening if an authTitle is set

Reply:
I've passed authflow options through to XboxTokenManager to make this possible, if this isn't acceptable we could pass an option through `verifyTokens(relayingParty, authTitle)` instead?